reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -677,16 +678,62 @@ public static void disableMetrics() {
 	/**
 	 * Re-apply default factory to {@link Schedulers}
 	 */
-	public static void resetFactory(){
+	public static void resetFactory() {
 		setFactory(DEFAULT);
 	}
 
+	/**
+	 * Set the current {@link Schedulers} aside in a capturing {@link Snapshot} that
```

Comment:
This should be the second sentence. The current javadoc doesn't mention that this allows setting the factory in the first place.